src/hardware/input/mouseif_dos_driver.cpp

```diff
@@ -96,12 +96,12 @@ static MouseButtons12S pending_button_state = 0;
 // These values represent 'hardware' state, not driver state
 
 static MouseButtons12S buttons = 0;
-static float pos_x             = 0.0f;
-static float pos_y             = 0.0f;
-static int8_t counter_w        = 0; // wheel counter
+static float pos_x = 0.0f;
+static float pos_y = 0.0f;
+static int8_t counter_w = 0; // wheel counter
 
-static bool use_relative    = true; // true = ignore absolute mouse position, use relative
-static bool is_input_raw    = true; // true = no host mouse acceleration pre-applied
+static bool use_relative = true; // true = ignore absolute mouse position
```

Comment:
Can you move the comments to the preceding lines?

Reply:
OK, changed.